docker/kites/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:latest
```

Comment:
Don't depend on the `latest` tag, otherwise the build is not reproducible.

Reply:
My mistake, I provide immediately